locales/en.default.schema.json

```diff
@@ -724,6 +736,24 @@
     "header": {
       "name": "Header",
       "settings": {
+        "color_scheme": {
+          "options__1": {
+            "label": "Accent 1"
```

Comment:
Not urgent: there's quite a few instances of these colours throughout Dawn and each setting has its own translations—we can clean this up by referencing the same translation.